types/dns2/index.d.ts

```diff
@@ -64,6 +64,8 @@ declare namespace DNS {
 
     interface DnsQuestion {
         name: string;
+        type: number;
```

Comment:
Might be nicer to refer to the given types e.g.

```suggestion
        type: Packet.TYPE;
```

(and same for class)

Reply:
Agreed, these fields should both use those existing types I think, unless there's a specific reason to be more vague.